pages/index.tsx

```diff
@@ -58,6 +57,21 @@ export default function Home() {
     setPlacesInfo(prevVal => prevVal.results)
   }
 
+  const callApi = async (point: string) => {
+    const params = {
+      city: 'Paris',
```

Comment:
[dust] Might wanna put a note here so we remember to make this a variable and not hardcoded
Im sure we will figure it out though